deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml

```diff
@@ -56,6 +56,12 @@ metadata:
       ]
     capabilities: Full Lifecycle
     categories: Storage
+    containerImage: quay.io/rhceph-dev/ocs-operator:4.3.0
```

Comment:
This is not correct. We should not be pointing at the downstream images from here. Only use images in `quay.io/ocs-dev`.